addons/cookoff/XEH_postInit.sqf

```diff
@@ -89,7 +83,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 
 // blow off turret effect
 ["Tank", "killed", {
-    if ((_this select 0) getVariable [QGVAR(enable),GVAR(enable)]) then {
+    if ((_this select 0) getVariable [QGVAR(enable), GVAR(enable)] !=0) then {
```

Comment:
It was that in a previous version, Commy advise me to change it as per https://github.com/acemod/ACE3/pull/7147#discussion_r317375977

Reply:
if it's set to false it will throw:
```
Error in expression <ce_cookoff_enable", ace_cookoff_enable] !=0) then {
if (random 1 < 0.15) then {
>
   Error position: <!=0) then {
if (random 1 < 0.15) then {
>
   Error !=: Type Bool, expected Number,String,Not a Number,Object,Side,Group,Text,Config entry,Display (dialog),Control,Network Object,Team member,Task,Location
 File z\ace\addons\cookoff\XEH_postInit.sqf..., line 86
```

Reply:
Can I get away with `if (!(_this select 0) getVariable [QGVAR(enable), GVAR(enable)] isEqualTo 0)` @PabstMirror